Example/Models/UserProfileModel.swift

```diff
@@ -67,6 +83,10 @@ func updateContactList(list: [MAContact]?) {
     }
 }
 
+func updateDeeplinkList(list: [String]?) {
+    _ = setDeepLinksList(deeplinksList: list)
```

Comment:
You should set the the result as discardable (`@discardableResult`)
```suggestion
    setDeepLinksList(deeplinksList: list)
```